src/util/console-color.ts

```diff
@@ -10,6 +10,8 @@ interface ConsoleState {
 }
 
 export class ColoredConsole {
+  public static blurSecrets = false;
```

Comment:
This doesn’t seem to be used ?

Reply:
It is not used in this class, but is used as a global variable set in  esphome-main and used in remote-process I see that is confusing - I'll move it to remote-process rather.
It started out there as I wanted to set the style in color-consol, but the styles were created too early so it didn't update